Phar/Phar.php

```diff
@@ -729,18 +729,18 @@ public function stopBuffering () {}
 	 * (PHP &gt;= 5.3.0, PECL phar &gt;= 2.0.0)<br/>
 	 * mapPhar for web-based phars. front controller for web applications
 	 * @link https://php.net/manual/en/phar.webphar.php
-	 * @param string $alias [optional] <p>
+	 * @param null|string $alias [optional] <p>
```

Comment:
Seems these types also can be updated in the method signature

Reply:
I am not sure if I should use ```?string $alias = null```or ```string $alias = null``` here?

Reply:
`?string $alias = null`. Please note that a similar change should be done for other parameters in the method

Reply:
ok